appveyor.yml

```diff
@@ -2,59 +2,35 @@ version: "{branch}-ci-{build}"
 os: Visual Studio 2015
 
 environment:
-  RUBY_VERSION: 200
   matrix:
-  - build: mingw
-    platform: x86
+  - CMAKE_GENERATOR: 'MSYS Makefiles'
     MINGW_ARCH: i686
-    MSYSTEM: MINGW32
     MINGW_ROOT: C:\msys64\mingw32
     BOOST_ROOT: C:\msys64\mingw32
     BOOST_LIBRARYDIR: C:\msys64\mingw32\lib
     BOOST_INCLUDEDIR: C:\msys64\mingw32\include\boost
-    CMAKE_GENERATOR: 'MSYS Makefiles'
-  - build: mingw
-    platform: x64
+    QT_DIR: C:\Qt\5.8\mingw53_32
+  - CMAKE_GENERATOR: 'MSYS Makefiles'
     MINGW_ARCH: x86_64
-    MSYSTEM: MINGW64
     MINGW_ROOT: C:\msys64\mingw64
     BOOST_ROOT: C:\msys64\mingw64
     BOOST_LIBRARYDIR: C:\msys64\mingw64\lib
     BOOST_INCLUDEDIR: C:\msys64\mingw64\include\boost
-    CMAKE_GENERATOR: 'MSYS Makefiles'
-  - build: msvc
+  - CMAKE_GENERATOR: 'NMake Makefiles'
     platform: x86
     BOOST_ROOT: C:\Libraries\boost_1_59_0
     BOOST_LIBRARYDIR: C:\Libraries\boost_1_59_0\lib32-msvc-14.0
     BOOST_INCLUDEDIR: C:\Libraries\boost_1_59_0\boost
-    CMAKE_GENERATOR: 'NMake Makefiles'
-  - build: msvc
+    QT_DIR: C:\Qt\5.8\msvc2015
+  - CMAKE_GENERATOR: 'NMake Makefiles'
     platform: x64
     BOOST_ROOT: C:\Libraries\boost_1_59_0
     BOOST_INCLUDEDIR: C:\Libraries\boost_1_59_0\boost
     BOOST_LIBRARYDIR: C:\Libraries\boost_1_59_0\lib64-msvc-14.0
-    CMAKE_GENERATOR: 'NMake Makefiles'
-
-install:
-- git submodule init
-- git submodule update
-- set PATH=C:\Ruby%RUBY_VERSION%\bin;%BOOST_LIBRARYDIR%;%PATH%
-- gem install bundle
-- bundle install
-- bundle env
```

Comment:
it is as much debugging by print as echo %path%. This is what it looks like on my machine:
$ bundle env
## Environment

```
Bundler   1.15.3
Rubygems  2.6.11
Ruby      2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
Git       2.14.1
Platform  x86_64-linux
OpenSSL   OpenSSL 1.1.0e  16 Feb 2017
```